website/content/en/preview/getting-started/getting-started-with-terraform/_index.md

```diff
@@ -131,29 +131,16 @@ module "vpc" {
 module "eks" {
   # https://registry.terraform.io/modules/terraform-aws-modules/eks/aws/latest
   source  = "terraform-aws-modules/eks/aws"
-  version = "18.31.0"
+  version = "~> 19.0"
 
   cluster_name    = local.cluster_name
   cluster_version = "1.24"
 
   vpc_id     = module.vpc.vpc_id
   subnet_ids = module.vpc.private_subnets
 
-  # Required for Karpenter role below
-  enable_irsa = true
-
-  node_security_group_additional_rules = {
-    ingress_nodes_karpenter_port = {
-      description                   = "Cluster API to Node group for Karpenter webhook"
-      protocol                      = "tcp"
-      from_port                     = 8443
-      to_port                       = 8443
-      type                          = "ingress"
-      source_cluster_security_group = true
-    }
-  }
 
-  node_security_group_tags = {
+  tags = {
```

Comment:
we don't want to apply this tag too broadly - the reason it works in the example linked is because for Fargate only clusters, the cluster primary security group is the only security group used. For this getting started guide that shows EKS managed node groups, lets leave this applied to only the node group `node_security_group_tags`

Reply:
thing is this method does not work in v19 of eks module. it doesnt apply the tag at all..

Reply:
I've just tested locally with v19 and the tags work as intended, this change is not warranted

Reply:
fixed